src/main/java/ttt/board/BoardFactory.java

```diff
@@ -0,0 +1,7 @@
+package ttt.board;
+
+public class BoardFactory {
```

Comment:
I introduced it so that in my Game test I could tdd in the user being prompted for the dimension. ie  I wanted to check that the board created was the size that the user had specified. I achieved this by having a BoardFactoryStub which caught the dimension added, which I could use in the test verification.